docs/components/movement-sensor/gps/set-up-base-station.md

```diff
@@ -0,0 +1,211 @@
+- **Mount Point:** [insert name of mountpoint from your [RTK2GO reservation](#reserve-an-rtk2go-casting-service)]
+- **Mount Point PW:** [insert password from your [RTK2GO reservation](#reserve-an-rtk2go-casting-service)]
+
+For example:
+
+{{< imgproc src="/components/movement-sensor/sparkfun-base-configuration.png" alt="Recommended Base Configuration." resize="900x" style="width:450px" >}}
+
+### Update WiFi configuration
+
+Select the **WiFi Configuration** dropdown on the RTK setup configure page.
+
+**Networks:** Enter credentials for up to 4 WiFi networks.
+The device tries all networks and uses the best network available.
+
+**Configure Mode:** WiFi.
+
+Your WiFi configuration should look like this, with the examples replaced with your real values:
+
+{{< imgproc src="/components/movement-sensor/sparkfun-wifi-configuration.png" alt="Example WiFi Configuration." resize="1000x" style="width:500px" >}}
+
+### Update Profile configuration
+
+Select the **Profile Configuration** dropdown on the RTK setup configure page.
+
+Select a profile to assign settings to that profile.
+At reset, your RTK device will use the selected settings profile.
+
+Your **Profile Configuration** should look like this:
+
+{{< imgproc src="/components/movement-sensor/sparkfun-profile-configuration.png" alt="Example Profile Configuration." resize="900x" style="width:450px" >}}
+
+### Save your configuration changes
+
+Select **Save Configuration** on the RTK setup configure page and wait for your configurations to take effect.
+
+## Start RTK2GO casting
+
+Using the reference kit's **MODE** button, switch the mode back to the default **Base**.
+Press the **MODE** button once to activate the menu.
+Press the button again to step through the available modes.
+Once highlighted, pause on **Base** to select this mode.
+
+Because you configured the base station to be in **Survey-in** mode in the [update Base configuration step](#update-base-configuration), it will take your base station up to six minutes to go into **Xmitting RTCM** mode.
+After a few minutes it goes into **Casting** mode.
+Once you see **Casting** on the mode menu display screen, go to `rtk2go.com:2101` in your browser and find your mount point in the source table.
+
+## Next steps: configure an RTK movement sensor
+
+Open the configuration instructions for your RTK-enabled movement sensor model:
+
+- [`gps-nmea-rtk-pmtk`](/components/movement-sensor/gps-nmea-rtk-pmtk/)
+- [`gps-nmea-rtk-serial`](/components/movement-sensor/gps-nmea-rtk-serial/)
+
+Follow the configuration instructions, providing the values you entered for your NTRIP server [Base configuration](#update-base-configuration) as the equal attributes in your Viam machine config:
+
+- `ntrip_url`: your RTK2GO casting URL, comprised of **Caster Host** and **Caster Port** (like `rtk2go.com:2101`)
+- `ntrip_username`: your **Caster User**, which should be your email
```

Comment:
Can we further specify that this email is the one that is used to setup the RTK reference station in http://rtk2go.com/sample-page/new-reservation/?